cmake/duktapeConfig.cmake

```diff
@@ -0,0 +1,33 @@
+# - Try to find duktape
```

Comment:
There should be some information where does this file originate

Reply:
It says so in the commit description, but sure, I'll happily include a link to [duktapeConfig.cmake.in](https://github.com/microsoft/vcpkg/blob/master/ports/duktape/duktapeConfig.cmake.in) here as well.

Reply:
Please do